apps/web/app/(app)/(survey-editor)/environments/[environmentId]/surveys/[surveyId]/edit/page.tsx

```diff
@@ -80,6 +80,7 @@ const Page = async ({ params }) => {
       attributeClasses={attributeClasses}
       responseCount={responseCount}
       membershipRole={currentUserMembership?.role}
+      organization={organization}
```

Comment:
You pass the whole `organization` to the frontend, even though you only need the `organizationId`. Please update to only using `organizationId`

Reply:
good point, fixed